tic-tac-toe-app/components/board.jsx

```diff
@@ -0,0 +1,72 @@
+const container = document.querySelector('.container')
+const game = new Game()
+
+const Board = () => {
+  const [box1, setBox1] = React.useState("")
+  const [box2, setBox2] = React.useState("")
+  const [box3, setBox3] = React.useState("")
+  const [box4, setBox4] = React.useState("")
+  const [box5, setBox5] = React.useState("")
+  const [box6, setBox6] = React.useState("")
+  const [box7, setBox7] = React.useState("")
+  const [box8, setBox8] = React.useState("")
+  const [box9, setBox9] = React.useState("")
+  
+  const move = (e) => {
+    
+    const boxStates = {
+      box1: setBox1,
+      box2: setBox2,
+      box3: setBox3,
+      box4: setBox4,
+      box5: setBox5,
+      box6: setBox6,
+      box7: setBox7,
+      box8: setBox8,
+      box9: setBox9,
+    }
+    const box = e.target.id
+    if(game.turn % 2 !== 0){
+      boxStates[box](<PlayerOne/>)
+      document.getElementById(box).removeAttribute("onClick");
+      console.log(box)
+      game.playerOne.makeMove(parseInt(box[3]) - 1)
+      console.log(game.board)
+      game.checkBoard(game.turn)
+      
+      game.turn += 1
+      return game.turn
+    }
+    if(game.turn % 2 === 0){
+      boxStates[box](<PlayerTwo/>)
+      document.getElementById(box).removeAttribute("onClick");
+      console.log(box)
+      game.playerTwo.makeMove(parseInt(box[3]) - 1)
+      console.log(game.board)
+      game.checkBoard(game.turn)
+      game.turn += 1
+      return game.turn
+    }
+  }
+  return (
+    <div>
+      <section className="row">
+        <div onClick={(e) => move(e)} className="box" id="box1">{box1}</div>
+        <div onClick={(e) => move(e)} className="box" id="box2">{box2}</div>
+        <div onClick={(e) => move(e)} className="box" id="box3">{box3}</div>
+      </section>
+      <section className="row">
+        <div onClick={(e) => move(e)} className="box" id="box4">{box4}</div>
+        <div onClick={(e) => move(e)} className="box" id="box5">{box5}</div>
+        <div onClick={(e) => move(e)} className="box" id="box6">{box6}</div>
+      </section>
+      <section className="row">
+        <div onClick={(e) => move(e)} className="box" id="box7">{box7}</div>
+        <div onClick={(e) => move(e)} className="box" id="box8">{box8}</div>
+        <div onClick={(e) => move(e)} className="box" id="box9">{box9}</div>
```

Comment:
If there were a refactor to use an array as a default state, then there would be an added benefit of being able to map through the boxes in order to render the board.

Reply:
Definitely a change I'll be making